scripts/test/Muon/results_tab_widget/results_tab_model_test.py

```diff
@@ -23,10 +24,9 @@
 
 
 def create_test_workspace(ws_name=None):
-    fake_ws = WorkspaceFactory.create('Workspace2D', 1, 1, 1)
     ws_name = ws_name if ws_name is not None else 'results_tab_model_test'
-    AnalysisDataService.Instance().addOrReplace(ws_name, fake_ws)
-    return fake_ws
+    Load("MUSR22725", OutputWorkspace=ws_name)
```

Comment:
I won't hold up merging this but does this really need to load data - it slows down the tests a lot?